src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -118,6 +100,14 @@ namespace compiler {
 namespace {
 #if defined(_WIN32) && !defined(__CYGWIN__)
 static const char* kPathSeparator = ";";
+// DO NOT include <io.h>, instead create functions in io_win32.{h,cc} and import
+// them like we do below.
+using google::protobuf::stubs::access;
```

Comment:
Sorry for the long silence. Done, though I moved them to `google::protobuf::internal::win32`, because the method names are generic enough that they might collide with similarly named functions.